tests/unit/btree_stress_test.c

```diff
@@ -234,23 +234,15 @@ CTEST2(btree_stress, test_random_inserts_concurrent)
    if (!iterator_tests(
           (cache *)&data->cc, &data->dbtree_cfg, root_addr, nkvs, data->hid))
    {
-      platform_default_log("invalid ranges in original tree\n");
+      CTEST_ERR("invalid ranges in original tree\n");
    }
 
-   /* platform_default_log("\n\n\n"); */
```

Comment:
I'm not so sure about deleting this dead code and the old lines on L249 onwards below.

Seems like the code author had felt a need that this kind of debugging would be handy if ever this one had to troubleshoot failures in this test's execution.

I'm generally tolerant about such code being left behind, as it does help some future engineer who might have to debug this stuff. (Rob didn't complain, surprisingly ... as he's the author.)

What if we (a) either retain this dead-code in comments or (b) enclose it under if (FALSE) { }? The code is there for troubleshooting and doesn't get in anyone's way.

Reply:
I strongly dislike dead code.  I'm going to delete it.  No one is speaking up to claim this code as theirs, and I'm guessing it could be added back by someone who's making changes.